sdk/eventhub/event-hubs/package.json

```diff
@@ -71,7 +71,7 @@
   },
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
-    "@azure/core-amqp": "^1.1.0",
+    "@azure/core-amqp": "^1.1.1",
```

Comment:
Changing the min version here is independent of the validation that this PR title is talking about.
Moving forward, lets have such changes in a separate PR.
Helps with git blaming in the future.

For example, in the future it wouldn't be clear as to why we decided to bump the min version here even if we git blame and look for a commit that made this change